packages/cli/test/config.mock.cjs

```diff
@@ -17,23 +20,16 @@ module.exports = {
           docsUrl: 'http://www.my-docs.dev',
         },
       ],
-      runner: {
-        command: 'node',
-        args: [
-          '-e',
-          `require('fs').writeFileSync('${outputDir}/out.json', '${JSON.stringify(
-            [
-              {
-                title: 'dummy-title',
-                slug: 'command-object-audit-slug',
-                value: 0,
-                score: 0,
-              },
-            ],
-          )}')`,
+      runner: createFileWriteRunnerConfig(
```

Comment:
we have mock functions in models for that situation living in the `test/fixture` folder

Reply:
Again, this is a function that is reusable for multiple tests (currently in cli and lighthouse plugin) and otherwise would need to be duplicated. In one of my new issues, I plan to address this by extracting reusable functions to a separate package instead of duplicating logic in multiple packages.